File: problem1/vers_java/P1.java
/*
 * P1.java
 *
 * Zeke Reyna
 *
 * Finds the sum of all multiples of 3 and 5 below 1000
 */

public class P1
{
        public static void main( String[] args )
        {
                int sum;
                int i;
                int bound;

                bound = 1000;

                sum = 0;
                for( i = 0; i < bound; i++ ) {
                        if( i % 3 == 0 || i % 5 == 0 )
                                sum += i;
                }

                System.out.print("\nThe sum of all multiples of 3 and 5 below");
                System.out.print(" "+bound+" is "+sum+"\n\n");
        }
}

File: problem1/vers_cpp/p1.cpp
/*
 * p1.cpp
 *
 * Zeke Reyna
 *
 * This function finds the sum of all multiples of 3 and 5 below 1000
 */

#include <iostream>
#include <cstdlib>

#define BOUND 10

using namespace std;

int main( int argc, char **argv )
{
	int sum;
	int i;
	int bound;

	(argc == 2) ? (bound = atoi( argv[1] )) : (bound = BOUND);

	sum = 0;
	for( i = 0; i < bound; i++ ) {
		if( i % 3 == 0 || i % 5 == 0 )
			sum += i;
	}

	cout << "\nThe sum of all multipes of 3 and 5 below " << bound;
	cout << " is "<< sum << "\n\n";

	return 0;
}
